### PR TITLE
OCPQE-23008 Output not recommended conditions for debugging

### PR DESCRIPTION
**Pull request by JianLi-RH**

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.14-gcp-ipi-basecap-none-f14

**Comment by JianLi-RH** [comment]
@shellyyang1989 Rerun the job passed: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53456/rehearse-53456-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.14-gcp-ipi-basecap-none-f14/1803709325298896896

/cc @jianlinliu @shellyyang1989 @jiajliu

**Comment by shellyyang1989** [comment]
> @shellyyang1989 Rerun the job passed: https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/53456/rehearse-53456-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-stable-4.16-upgrade-from-stable-4.14-gcp-ipi-basecap-none-f14/1803709325298896896
> 

Did the job run on the same version as the original failed one?

**Comment by JianLi-RH** [comment]
@shellyyang1989 Versions in this job:

```
INFO[2024-06-20T08:40:13Z] Importing release 4.16.0-rc.6 created at 2024-06-18 17:43:01 +0000 UTC with 190 images to tag release:target ... 
INFO[2024-06-20T08:40:15Z] Importing release 4.15.18 created at 2024-06-13 09:07:26 +0000 UTC with 191 images to tag release:intermediate415 ... 
INFO[2024-06-20T08:40:17Z] Importing release 4.14.30 created at 2024-06-13 11:18:33 +0000 UTC with 189 images to tag release:latest ... 
```
Versions in the failed job:
```
INFO[2024-06-16T13:24:45Z] Importing release 4.16.0-rc.4 created at 2024-06-07 14:09:04 +0000 UTC with 190 images to tag release:target ... 
INFO[2024-06-16T13:26:12Z] Imported release 4.15.18 created at 2024-06-13 09:07:26 +0000 UTC with 191 images to tag release:intermediate415 
INFO[2024-06-16T13:26:12Z] Imported release 4.14.29 created at 2024-06-06 14:16:22 +0000 UTC with 189 images to tag release:latest 
```

**Comment by shellyyang1989** [comment]
ok, no concern from my side regarding outputting the var. Please change the version back and fix the ci-operator-config-metadata issue

File: ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh [hunk 1]
```diff
@@ -636,6 +636,8 @@ function upgrade() {
     local retry=3 unrecommened_conditional_updates
     while (( retry > 0 )); do
         unrecommened_conditional_updates=$(oc get clusterversion version -o json | jq -r '.status.conditionalUpdates[]? | select((.conditions[].type == "Recommended") and (.conditions[].status != "True")) | .release.version' | xargs)
+        echo "Not recommended conditions: "
+        echo "${unrecommened_conditional_updates}"
```
Comment (jhou1) [inline ci-operator/step-registry/cucushift/chainupgrade/toimage/cucushift-chainupgrade-toimage-commands.sh:640]: The variable should probably be renamed to `unrecommended_conditional_updates`. 

unrecommen***d***ed

**Pull request merged**